src/taskProvider.ts

```diff
@@ -86,18 +86,18 @@ export class TaskProvider implements vscode.TaskProvider {
     },
     {
       group: "Build",
-      name: "Build",
+      name: "Build incremental",
       builders: { ["C"]: this.cBuildExec, ["Rust"]: this.rustBuildExec },
-      toolTip: "Build app in release mode",
+      toolTip: "Build app (incremmenatal mode)",
```

Comment:
What does `incremmenatal` mean ?

Reply:
Fixed the typo :wink:

Reply:
I still see `incremmenatal`, you probably did not push yet :sweat_smile: